view-generator-framework/build.gradle.kts

```diff
@@ -12,19 +12,19 @@ tasks.test {
 
 dependencies {
   implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.64")
-  implementation("org.hypertrace.core.kafkastreams.framework:kafka-streams-framework:0.4.3")
-  implementation("org.hypertrace.core.kafkastreams.framework:avro-partitioners:0.2.13")
-  implementation("org.apache.avro:avro:1.11.1")
+  implementation("org.hypertrace.core.kafkastreams.framework:kafka-streams-framework:0.4.5")
+  implementation("org.hypertrace.core.kafkastreams.framework:avro-partitioners:0.4.5")
+  implementation("org.apache.avro:avro")
```

Comment:
Is the version derived from some boms coming from a transitive dep?

Reply:
Yeah, the kafka streams frameworks is meant to define all the kafka and avro versions, and uses a bom to do so.